scripts/perftune.py

```diff
@@ -1294,6 +1317,7 @@ def names():
 argp.add_argument('--options-file', help="configuration YAML file")
 argp.add_argument('--dump-options-file', action='store_true', help="Print the configuration YAML file containing the current configuration")
 argp.add_argument('--dry-run', action='store_true', help="Don't take any action, just recommend what to do.")
+argp.add_argument('--write-back-cache', help="Enable/Disable \'write back\' write cache mode.", dest="set_write_back")
```

Comment:
you can make the default type to _distutils.util.strtobool_ and you won't need to check types after that.
```suggestion
argp.add_argument('--write-back-cache', dest="set_write_back", type=distutils.util.strtobool, default=False, help="Enable/Disable \'write back\' write cache mode.", )
```

Reply:
The whole point is that I don't want to.
Please, read the description of the patch. There is a special meaning when the value is not set: don't change the currently configured write cache configuration.

Defining a default value as you suggested would have changed the desired semantics.

Reply:
@bentsi Please, let me know if that makes more sense now?